milestones/models.py

```diff
@@ -94,7 +96,7 @@ class Meta:
         unique_together = (("course_id", "milestone"),)
 
     def __unicode__(self):
-        return unicode("%s:%s:%s" % (self.course_id, self.milestone_relationship_type, self.milestone))
+        return str("%s:%s:%s" % (self.course_id, self.milestone_relationship_type, self.milestone))
```

Comment:
This `str` seems unneeded with the string-formatting that's happening here? The three parameters will already be string-encoded irregardless of their type.